### DISCUSSION TITLE
Preserve file modification time in fix mode with option --preserve-mtime

### DISCUSSION THREAD

**Discussion by dinetxum** [PHP-CS-Fixer/PHP-CS-Fixer#8144] [Feature ideas]
### Feature request

Preserve  file modification time in fix mode with new option --preserve-mtime

```
$file_to_be_modified =  '/fixed/file.php';
$old_modification_timestamp = filemtime($file_to_be_modified);
...
touch($file_to_be_modified,$old_modification_timestamp);

```

**Comment by Wirone** [comment]
@dinetxum why do you want Fixer to _not_ change modification time of the file when applying fixes? The files are modified, it's totally expected `mtime` is changed. What's your scenario?
